package.json

```diff
@@ -75,7 +85,8 @@
     "dotenv": "5.0.0",
     "dotenv-expand": "4.2.0",
     "electron": "2.0.1",
-    "electron-packager": "12.1.0",
+    "electron-builder": "20.28.4",
+    "electron-log": "^2.2.17",
```

Comment:
Nit: Version not pinned. 
Feel free to remove the `^` but I think we could also fix this after merging.

Reply:
Yeah, I'm not too worried.